src/20240320_AaveV2Ethereum_EthereumV2ReserveFactorAdjustment/EthereumV2ReserveFactorAdjustment.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Ethereum v2 Reserve Factor Adjustment"
+author: "Karpatkey, TokenLogic, ChaosLabs"
+discussions: "https://governance.aave.com/t/arfc-ethereum-v2-reserve-factor-adjustment/16764/3"
+---
+
+## Simple Summary
+
+This AIP is a continuation of proposal 49 on Governance v3 and increases the Reserve Factor (RF) for assets on Ethereum v2 by 5.00%, up to a maximum of 99.99%.
+
+## Motivation
+
+This AIP will reduce deposit yield for assets on Ethereum v2 by increasing the RF. With this upgrade being passed, users will be further encouraged to migrate from Ethereum v2 to v3.
+
+Increasing the RF routes a larger portion of the interest paid by users to Aave DAO's Treasury. User's funds are not at risk of liquidation and the borrowing rate remains unchanged.
+
+Of the assets with an RF set at 99.99%, there is no change. All other asset reserves will have the RF increased by up to 5%.
+
+## Specification
+
+The following parameters are to be updated as follows:
+
+| Asset | Previous Reserve Factor | New Reserve Factor |
+| ----- | ----------------------- | ------------------ |
+| DAI   | 25.00%                  | 30.00%             |
```

Comment:
DAI reserve factor is already at 30%, updated by this [AIP](https://vote.onaave.com/proposal/?proposalId=49&ipfsHash=0xd0254523d65549ea7d020d91df2b7905a27da988ecc8263f2cf2709c2e369cc7)

Do you wish to update it to 35%?